src/components/NextButton/NextButton.tsx

```diff
@@ -9,7 +11,13 @@ const StyledButton = styled(Button)`
   margin-right: 15px !important;
 `
 
-const NextButton = ({ className, children, onClick }) => (
+interface NextButtonProps {
+  className?: string
+  children: ReactNode
+  onClick: (event: BaseSyntheticEvent) => void
+}
+
+const NextButton = ({ className, children, onClick }: NextButtonProps) => (
   <StyledButton
     className={className}
     data-testid="next-button"
```

Comment:
Using typescript makes proptypes obsolete. So you can remove the proptypes. But, there is also a defaultProp, you should implement that in your change as well. 
Also, I should just name it Props, not NextButtonProps. It's only used in this component's context, adding the name is a bit double.

Reply:
I've removed the .prototypes, but according to the internet there is a difference between them. https://stackoverflow.com/questions/41746028/proptypes-in-a-typescript-react-application.

Done the other two.

Reply:
They are a different thing indeed, but in this ADR 0007-typescript.md we agreed to remove them in favor of typescript.